scripts/tagRelease.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Not tagging if HEAD is already tagged
+if git name-rev --tags --name-only --no-undefined HEAD &>/dev/null; then
+	read -p "Your latest commit is tagged. Do you really want to re-tag? [yN] " choice
+	case $choice in
+		[Yy] ) :;;
+		* ) exit;;
+	esac
+fi
+
+# Parsing current tag
+
+# Using name-rev is not good because it only gives you one tag, and not
+# necessary the latest version
+today_tag="$(date --rfc-3339=date)"
+latest_today_tag="$(git tag | grep "$today_tag" | sort | tail -1)"
```

Comment:
Note that this will (I think) fail with number 10, as that would sort before 2 :-) Not a problem in this case I think.